src/components/checkboxes/template.test.js

```diff
@@ -255,6 +255,36 @@ describe('Checkboxes', () => {
     })
   })
 
+  it('render hint text', () => {
+    const $ = render('checkboxes', {
```

Comment:
I figure that checking for input and hint relationship are correct to that it gets announced.
I could just test that it renders, but that not that useful.

Reply:
It's definitely useful to test that, but there's still a lot going on here for one test:

- You could take out a lot of the lines here (for example the entire first item) and the test would still pass, which makes it harder for others coming in to try and understand what's _actually being tested_.
- There are three assertions being made, two of which aren't covered by the test description. If this test were to fail because the `aria-describedby` isn't what we expect, the description doesn't help you to understand why it's failing.

If you look at e.g. https://github.com/alphagov/govuk-frontend/blob/master/src/components/input/template.test.js#L85-L115:

- there's only one assertion per test
- the description of each test describes the assertion that's being made
- we're asserting on the behaviour we actually care about – we don't actually care what the `id` or `aria-describedby` values are – as long as the id is included in `aria-describedby`. For example, if we added another element which we wanted to add to `aria-describedby` then this test as written would break (incorrectly) and we'd have to come back and fix it.
- there's very little you could delete from each test without breaking it, so it's clear what the test is doing.

Does that make sense?

Reply:
yep. i've updated the tests